.github/workflows/doc-build.yml

```diff
@@ -30,8 +30,8 @@ jobs:
     - name: install-pip
       run: |
         pip3 install setuptools
-        pip3 install 'breathe>=4.9.1,<4.15.0' 'docutils>=0.14' \
-                     'sphinx>=1.7.5,<3.0' sphinx_rtd_theme sphinx-tabs \
+        pip3 install 'breathe>=4.20.0' 'docutils>=0.16' \
```

Comment:
maybe just use `pip3 install -r scripts/requirements-doc.txt`, this way we do not have to maintain the versions twice and get out of sync.

Reply:
done!